src/workloads/execution/UnshardedLookup.yml

```diff
@@ -1,31 +1,54 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  Benchmark a $lookup from one unsharded collection to another in a sharded environment.
+  Benchmark a $lookup where the foreign collection (right side) is unsharded.
```

Comment:
nit: Could you change this to "Benchmark a $lookup in a sharded environment where ..."? It would help with potential confusion if a reader was wondering why we we're running even unsharded-unsharded workloads with sharding enabled.